pkgs/development/python-modules/fcmaes/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib, buildPythonPackage, cmake, fetchPypi, python, armadillo, scipy, mkl }:
+
+buildPythonPackage rec {
+  pname = "fcmaes";
+  version = "0.9.5.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1za05afp6x98dmdj61brkkdfzkgv0ixyafp8zh94fa001vs2qrvl";
+  };
+  
+  dontUseCmakeConfigure = true;
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  preConfigure = ''
+    sed -i '/clang/d' _fcmaescpp/CMakeLists.txt
+    sed -i 's!../fcmaes!../fcmaes/lib!' _fcmaescpp/CMakeLists.txt
+    sed -i 's/sudo//' _fcmaescpp/install.sh
+  '';
+
+  preBuild = ''
+    cd _fcmaescpp
+    sh install.sh
+    cd ..
+  '';
+
+  propagatedBuildInputs = [
+    armadillo
+    mkl
```

Comment:
is mkl a runtime dependency or build dependency? I would guess build, in which case it should be in buildInputs.

Reply:
It's a runtime dependency: `mkl_rt` is loaded when we import `fcmaescpp` in python.